pkg/matcher/annotation_matcher.go

```diff
@@ -24,6 +24,10 @@ const (
 	// regex allows array of string or a single string
 	// eg. ["foo", "bar"], ["foo"] or "foo".
 	reValidateTag = `^\[(.*)\]$|^[^[\]\s]*$`
+
+	// MaxCommentLogLength is the maximum number of characters
+	// to display in logs for gitops comments.
+	MaxCommentLogLength = 160
```

Comment:
nitpick: if this field isn't configurable, I don't think it should be public.

Reply:
can you do that change please so we can merge this? thanks

Reply:
I think my comment about configurability and applying it at the log level may have delayed resolving this comment. Making the variable private is the only suggestion which I think should be applied in this PR. Logger-level max-length and configuration are almost certainly scope creep and I should have probably brought them up in another communication channel. Sorry if that introduced any confusion or delays

Reply:
I've implemented the requested changes and explored the possibility of logger-level truncation. With further brainstorming, I believe we can refine this approach and come up with a solution for an option to disable and/or change the max log length.